src/npr/ont_pipeline.py

```diff
@@ -172,7 +177,15 @@ def read_flowcell_info(config, info_dict, base_path):
         info_dict["flowcell"] = jsondata['protocol_run_info']['meta_info']['tags']['flow cell']['string_value']
         info_dict["kit"] = jsondata['protocol_run_info']['meta_info']['tags']['kit']['string_value']
         info_dict['barcoding'] = bool(jsondata['protocol_run_info']['meta_info']['tags']['barcoding']['bool_value'])
-        info_dict['model_def'] = jsondata['protocol_run_info']['meta_info']['tags']['default basecall model']['string_value']
+        # check run paramenters to capture model
+        model = None
+        for par in jsondata['protocol_run_info']['args']:
+            if par.startswith('--guppy_filename='):
+                px,model = par.split("=")
+                info_dict['model_def'] = model
```

Comment:
This should be accompanied with changes in 291 - 299 and/or snakehelper.guppy2dorado, as I understand now info_dict['model'] is not necessarily the same as what you have here.

Reply:
true, I need to dig more into this